content/source/docs/enterprise/sentinel/import/tfstate.html.md

```diff
@@ -169,31 +169,14 @@ main = rule { tfstate.module_paths contains ["foo"] }
 
 -> Note the above example only applies if the module is present in the state.
 
-#### Iterating through modules
+#### Iterating Through Modules
 
 Iterating through all modules to find particular resources can be useful. This
-example shows how to use `module_paths` with the [`module()`
-function](#function-module-) to retrieve all resources of a particular type from
-all modules (in this case, the [`azurerm_virtual_machine`][ref-tf-azurerm-vm]
-resource). Note the use of `else []` in case some modules don't have any
-resources; this is necessary to avoid the function returning undefined.
+[example](iterate-over-modules) shows how to use `module_paths` with the
+[`module()` function](#function-module-) to find all resources of a
+particular type from all modules using the `tfplan` import. By changing `tfplan` in this function to `tfstate`, you could make a similar function find all resources of a specific type in the current state.
```

Comment:
Please wrap this line to 79 characters to match the rest of the document.

Reply:
ok